Server/JokesOnYou.Web.Api/appsettings.Development.json

```diff
@@ -2,7 +2,7 @@
   "Logging": {
     "LogLevel": {
       "Default": "Information",
-      "Microsoft": "Warning",
+      "Microsoft": "Information",
```

Comment:
why these changes ?

Reply:
So that we get more logs on the console as to what's happening, easier to debug

Reply:
> why these changes ?

By changing we see: SQL statements made to the database and Http requests to our api etc..